tests/KeyFrame.cpp

```diff
@@ -506,3 +513,195 @@ TEST_CASE( "Point_Vector_Constructor", "[libopenshot][keyframe]" )
 	CHECK(k1.GetLength() == 11);
 	CHECK(k1.GetValue(10) == Approx(30.0f).margin(0.0001));
 }
+
+TEST_CASE( "TrackedObjectBBox init", "[libopenshot][keyframe]" )
+{
+	TrackedObjectBBox kfb;
+	// XXX: This really needs to perform some sort of _test_ here,
+	//      like confirming some default values in the new object.
+}
```

Comment:
As the comment says, this test case really needs to _test_ something, for there to be any point to it. Otherwise, all of the _other_ test cases that start off by creating a default `TrackedObjectBBox` object are already performing the same test, making this redundant.